### PR TITLE
[ADD] mrp_analytic_cost

### PR DESCRIPTION
**Pull request by dreispt**
Supersedes #575.
Merges the previous `mrp_analytic_cost_operation`and `mrp_analytic_cost_operation` into a single module, `mrp_analytic_cost`.

### CODE REVIEW

**Comment by dreispt** [comment]
@Chandresh-SerpentCS @patrickrwilson Please review

File: mrp_analytic_cost/views/mrp_workcenter_view.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline mrp_analytic_cost/views/mrp_workcenter_view.xml:1]: This line is no longer needed (since v11)

**Review by patrickrwilson — APPROVED** [review]
Very small change, no blockers. LGTM.

**Pull request merged**